### PR TITLE
revert names to filter/converter

### PR DESCRIPTION
**Pull request by yoouung**
- according to the results of 1017 datatree meeting
  - custom filter -> `filter`
  - auto filter -> `converter`

### CODE REVIEW

File: dt-demo-fe/src/pages/02_filter.js [hunk 1]
```diff
@@ -10,12 +10,12 @@ const MainContainer = styled.div`
...
+const Filter = () => {
     return (
         <MainContainer>
             <Outlet />
         </MainContainer>
     );
 };
 
-export default AutoFilter;
+export default Filter;
```
Comment (doorooful) [inline dt-demo-fe/src/pages/02_filter.js:21]: Can you check if the file name matches?

File: dt-demo-fe/src/pages/03_converter.js [hunk 2]
```diff
@@ -10,12 +10,12 @@ const MainContainer = styled.div`
...
+const Converter = () => {
     return (
         <MainContainer>
             <Outlet />
         </MainContainer>
     );
 };
 
-export default CustomFilter;
+export default Converter;
```
Comment (doorooful) [inline dt-demo-fe/src/pages/03_converter.js:21]: Same here

**Review by doorooful — COMMENTED** [review]
Please check according to my comments 😃
Also can you manually verify if the program works as intended? I think it'd be better to attach screenshot here.

**Comment by yoouung** [comment]
### UI

1. home
Screenshot 2023-10-21 at 12 23 35 AM

2. filter
Screenshot 2023-10-21 at 12 23 40 AM

3. converter
Screenshot 2023-10-21 at 12 23 45 AM

4. statistics
Screenshot 2023-10-21 at 12 25 06 AM

**Comment by yoouung** [comment]
Filter works well!
- detection
image
- conversion
image

---
Converter works well!
image

**Review by doorooful — APPROVED** [review]
Thank you!

**Pull request merged**